### PR TITLE
Bump CKE4 integrations and dependencies

### PR DESCRIPTION
**Pull request by Dumluregn**
The main objective was to update our integrations with Angular, React and Vue packages. I also bumped some dependencies to ensure their versions match the ones in the according integrations repos.

Closes https://github.com/ckeditor/ckeditor4/issues/4159.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
     "babel-loader": "^8.0.4",
     "chalk": "^2.4.1",
     "cheerio": "^1.0.0-rc.2",
-    "ckeditor4-angular": "^1.0.0",
+    "ckeditor4-angular": "^1.3.0",
     "ckeditor4-plugin-spreadsheet": "^1.2.0",
-    "ckeditor4-react": "^1.0.0",
-    "ckeditor4-vue": "^0.1.0",
+    "ckeditor4-react": "^1.2.0",
+    "ckeditor4-vue": "^1.1.0",
```
Comment (f1ames) [inline package.json:26]: Interesting, in my local docs copy I had latest integrations version, while e.g. `^0.1.0` for Vue shouldn't allow it...

**Review by f1ames — CHANGES_REQUESTED** [review]
Angular demo stopped working in IE11 :thinking: It works fine in the latest online docs, however it might be good to check if it works too with `stable` build locally.



Since you have aligned the deps with our Angular integration I wonder it it's some docs issue or integration issue itself :thinking:

File: package.json (hunk 1 above)
Comment (f1ames) [reply package.json:26]: :point\_up: Just a random thought, nothing to do here :stuck\_out\_tongue:

Comment (f1ames) [reply package.json:26]: :point\_up: Just a random thought, nothing to do here :stuck\_out\_tongue:

Comment (Dumluregn) [reply package.json:26]: Not sure why but also I had `ckeditor4-angular 1.2.2` installed although `^1.0.0` should install version `1.3.0`. Not sure if I misunderstood the idea or something doesn't work well 🤔

**Comment by Dumluregn** [comment]
> Angular demo stopped working in IE11 🤔 It works fine in the latest online docs, however it might be good to check if it works too with stable build locally.

I suppose there might be some files missing since the docs project structure is a bit different than in integration repo. Thanks for catching it, I'll check it out.

**Comment by Dumluregn** [comment]
As for the IE11 - I didn't manage to make it work with all the dependencies I tried to update, so I resigned from some of them. Demo in the integration repo works fine, so it it's the matter of just docs sample. The difference between the two is that in docs we are using webpack at the end, while in angular we don't.

Please note that one error (concerning the `closest()` method) will probably still be present in the console. However from what I see it is also there when current `stable` branch is built locally, so nothing changes here.

**Review by f1ames — APPROVED** [review]
LGTM :+1: 

---

> As for the IE11 - I didn't manage to make it work with all the dependencies I tried to update, so I resigned from some of them. Demo in the integration repo works fine, so it it's the matter of just docs sample. The difference between the two is that in docs we are using webpack at the end, while in angular we don't.

Well, on one side it's good that our docs support IE11 (since CKEditor 4 supports it too), OTOH it's kind of limiting. Anyway, as long as the sample works fine with older deps and is compatible with integration itself I think we are fine.

> Please note that one error (concerning the `closest()` method) will probably still be present in the console. However from what I see it is also there when current `stable` branch is built locally, so nothing changes here.

I see this error in Vue sample on production. In this PR is in React/Angular sample :thinking: Still it's IE11 only and everything seems to be working fine so let's just ignore it.

**Pull request merged**